recipes/strong_type/all/conanfile.py

```diff
@@ -23,6 +25,8 @@ def package_id(self):
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, 14)
+        if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "7":
```

Comment:
Is this only limited to the the newer version? the old one passed in previous PRs ?

Reply:
Yes. Previous versions did not trigger that gcc bug.  
But it's not worth keeping that combination alive for old compilers and adding branches in that recipe.
Nobody using that lib is tied to such old compilers. That's a different scene, so a simpler recipe wins. (IMHO)